docs/docs/authenticate/README.md

```diff
@@ -105,15 +39,22 @@ to add documentation for your provider.
 If you require assistance in setting up your OIDC provider, please
 join the [Firezone Slack group](https://www.firezone.dev/slack).
 
+### Pretty URLs
```

Comment:
This is a bit inaccurate. We don't generate a pretty URL -- the auth provider key is used in the URL. The key name can be any arbitrary string but needs to be unique.

If the user wishes to use `sjfqwo-sadfjs1213` for example, that would be the key used in the URL.